public/locales/en-US/translation.json

```diff
@@ -1,9 +1,36 @@
 {
   "dashboard": "Dashboard",
   "agreements": "Agreements",
+
```

Comment:
this file is starting to get a mess? Let's try to make it more organized by contexts? 
global terms could be on the root level, but screen specific could follow some rule.
Also, make keys alphabetically ordered, would be much better